### ISSUE TITLE
Field/option for battery manufacturer/series

### ISSUE THREAD

**Issue by we5** [andrew-codechimp/HA-Battery-Notes#159]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/andrew-codechimp/ha-battery-notes/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).

### Is your feature request related to a problem? Please describe.

Depending on the type of device and their power usage, in my testing sometimes the operation duration highly depends on the manufacturer and/or specific variants of the same type of batteries.

I tend to play around and test different battery series to see how they behave and my findings are, that some are lasting way longer than others. Even from the same brand, the different series can vary a lot. So, for example, the industrial AA from Varta do indeed last longer in some scenarios than their ultra power sisters.

### Describe the solution you'd like

The ultimative end goal for a feature I have in mind is a database of batteries (in general/ and specific to decides) and their actual usage.

To be more accurate I even consider looking at usage metrics, so I think a contact sensor could actually measure „contact changes“ while motion sensors could measure motion triggers. A TRV could measure valve reports and regular sensors updates to their measured values. As fallback a generic count of updates could also help to better knowledge about the efficiency of batteries and devices.

### Describe alternatives you've considered

Long time ago i started a spreadsheet but this was abandoned really fast because the time this took. And because with the former system (not HASS) it was not easy to implement (for me)

### Additional context

I am open to feedback on this, as it could be, that I am the only one that is annoyed by the variance of efficiency of the devices on the market and the claims from the brands are mostly just garbage. 

I think also only the simplest type of usage information could already help for beginners and all users that try to find a good sensor or battery powered device.

For example i am currently replacing all Aeotec sensors because they just suck batteries (CR123) like no others while the Philips sensors are the most efficient sensors I've tested so far. And they are using simple AAAs, which is also a factor in terms of maintenance costs.

**Comment by andrew-codechimp** [comment]
I'm still giving this some thought. It's pretty niche and won't appeal to everyone so I'd have to make it an advanced feature that doesn't get in the way of adding a battery note.
Current thinking is an attribute that's only editable via the configure button rather than at first add of a device.

**Comment by we5** [comment]
You are right, this is an absolute edge case and only for freaks. In the former home automation system (FHEM) I had exactly that, just a fuel for the manufacturer and battery series. Also I think it will not benefit from the beginning, having such information on the long run might give some more insights.

I am fine if you give this a low prio or nice-to-have label. I am already loving the battery notes and look forward to enhance battery level graphs with the info about battery replacement.

thanks!

**Comment by ChristophCaina** [comment]
> having such information on the long run might give some more insights.
at least in combination with the information when the battery (of each manufacturer) has been replaced.

In this case you could generate statistics ... but you would need a way to collect and link those information, if you really want to generate some kind of report out of the information...

I think, more interesting would be to know, if it is a (normal) battery or a rechargable one... because AA Batteries are usually 1.5 V and rechargable AA Batteries are 1.2 V -> wich can make a huge difference how long they remain in a particular device.

**Comment by we5** [comment]
That would be also nice to have for data nerds. In fact the Aeotec sensors are actually not compatible with rechargeables according to their manuals, but I tried it it anyways, because they are sucking the regular CR123 so fast, i couldn’t believe it. And it turns out, the rechargeable Fenix alternatives were not as bad as assumed. But they tend to calculate the remaining capacity completely wrong. Same with Fibaro sensors running on rechargeables. They all just have really bad battery managements and Aeotec is the worst.

That’s why I am so interested in general analytics for these devices from real world scenarios. But it’s hard to find.

**Comment by bmos** [comment]
I'd point out that batteries (especially lithium) are very temperature sensitive so battery lifespan data for devices without temp sensors is likely to be VERY noisy.

**Comment by andrew-codechimp** [comment]
Moving this to discussion/ideas so it's not lost but I'm not going to work on this in the very near future.
